README.md

```diff
@@ -1,4 +1,41 @@
 # supportkit-cordova
 Cordova (PhoneGap) bindings for SupportKit
 
-Still experimental - not yet officially supported. Feel free to hack on this.
\ No newline at end of file
+# Basic Setup
+
+## Init SupportKit with appToken
+
+
+	SupportKit.init({
+    	"appToken" : "your\_supportkit\_app\_token",
+    	"conversationAccentColor" : "#FFFFFF" // optional, default is #00B0FF
+	});
+
+See [http://docs.supportkit.io/api/Classes/SKTSettings.html](http://docs.supportkit.io/api/Classes/SKTSettings.html) for other properties you can set. 
+
+## Show SupportKit
+
+	SupportKit.show();
+
+# Awesome Setup
+
+## Add user info
+
+	SupportKit.setUser({
+		"firstName" : "Erlich",
+		"lastName" : "Bachman",
+		"email" : "erlich.bachman@hbo.com",
+		"signedUpAt" : 1397606400 // unix timestamp
+	});
+
+## Add custom user properties
+
+	SupportKit.setUserProperties({
+		"tv_show" : "Silicon Valley",
+		"episodes" : ["Server Space, "The Lady", "Bad Money"],
```

Comment:
Note that we don't explicitly support user properties as arrays, so it might not be a good idea to showcase an array here. If you set an array as a user prop, it will be converted into a string using `[theArray description]`.